.babelrc

```diff
@@ -25,6 +25,13 @@
         "transform-decorators-legacy"
       ]
     },
+    "test": {
+      "presets": [ "es2015-node4", "flow"],
+      "plugins": [
+        "transform-class-properties",
```

Comment:
Why not use `preset-stage-3` and `preset-stage-2`?

Reply:
The reason is we still use `^` in some dependencies. I like to have the control of which plugins we use. I think we can stably later on a final babel configuration using `env`. That would be after this huge merge. Are you agreed?

Reply:
Got it